scripts/movie-card.js

```diff
@@ -0,0 +1,104 @@
+const movie = {
+  id: 1,
+  title: "Interstellar",
+  description:
+    "The adventures of a group of explorers who make use of a newly discovered wormhole to surpass the limitations on human space travel and conquer the vast distances involved in an interstellar voyage.",
+  movie_year: 2014,
+  director: "Christopher Nolan",
+  actors: [
+    "Matthew McConaughey",
+    "Anne Hathaway",
+    "Jessica Chastain",
+    "Michael Caine",
+    "Casey Affleck",
+    "Mackenzie Foy",
+    "John Lithgow",
+    "Ellen Burstyn",
+    "Matt Damon",
+  ],
+  poster_url:
+    "https://www.themoviedb.org/t/p/w600_and_h900_bestv2/gEU2QniE6E77NI6lCU6MxlNBvIx.jpg",
+  price: "120.00",
+};
+document.addEventListener("DOMContentLoaded", function () {
```

Comment:
Your `<script>` is at the end of the `<body>`, so you should not have to wait for this event. Maybe remove the `type="module"` attribute if it does not work.